api/RTCRtpContributingSource.json

```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "RTCRtpContributingSource": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCRtpContributingSource",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
```

Comment:
59 for all three flavors of Chrome and for `RTCRtpContributingSource.timestamp` and `RTCRtpContributingSource.source`.